Azure dbs (#355)
Amsterdam/dso-api: dev-docs/source/environment.rst

This value defaults to ``AZURE_APPI_CONNECTION_STRING`` if not defined.

> Database configuration is fetched from environment variables while ``PGPASSWORD`` will be fetched from secret named ``mdbdataservices-read`` in KeyVault:
> 
> - ``PGDATABASE``
> 
> Database Name.
> 
> - ``PGUSER``
> 
> Database User Name.
> 
> - ``PGHOST``
> 
> Database Host Name.
> 
> - ``PGPORT``
> 
> Database Host Port.
> 
> If ``CLOUD_ENV`` is not set or not starting with ``azure`` is assumes its value to be a regular hosting provider.

Reviewer: ```suggestion
Database configuration is fetched from the environment, while ``PGPASSWORD`` will be fetched from the secret named ``mdbdataservices-read`` in KeyVault:

- ``PGDATABASE``

Database name.

- ``PGUSER``

Database username.

- ``PGHOST``

Database hostname.

- ``PGPORT``

Database host port.

If ``CLOUD_ENV`` is not set or does not start with ``azure``, it is assumed to be a regular hosting provider.
```
